package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "private": true,
+  "scripts": {
+    "prepare": "lerna run prepare",
+    "rebuild:browser": "theia rebuild:browser",
+    "rebuild:electron": "theia rebuild:electron"
+  },
+  "devDependencies": {
+    "lerna": "2.4.0"
+  },
+  "workspaces": [
+    "github-extension",
+    "browser-app",
+    "electron-app"
+  ],
+  "dependencies": {
+    "github": "^13.1.0"
```

Comment:
As far as I see this is the deprecated package https://www.npmjs.com/package/github ?

Reply:
Created a new CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16012

Reply:
CQ is approved, reworked to use newer package.